### ISSUE TITLE
3.21: Installed header includes non-existent header exports.h

### ISSUE THREAD

**Issue by yurivict** [grame-cncm/libmusicxml#57]
```include/libmusicxml/libmusicxml.h``` file has ```#include "exports.h"``` but  ```exports.h``` isn't installed and it isn't installed by any dependencies.

3.21 installs only these 2 headers:
```
include/libmusicxml/libmusicxml.h
include/libmusicxml/reader.h
```

**Comment by dfober** [comment]
should be fixed with the new version (3.22)

**Comment by yurivict** [comment]
This is fixed now.

Thanks!
